nvflare/fuel/f3/drivers/uds_driver.py

```diff
@@ -125,3 +124,13 @@ def get_socket_path(params: dict) -> str:
             socket_path = "/" + socket_path
 
         return socket_path
+
+    @staticmethod
+    def remove_socket_file(socket_path: str) -> bool:
+        try:
+            os.unlink(socket_path)
+        except OSError as error:
+            log.debug(f"Removing {socket_path} : {error}")
```

Comment:
This log here, should we say `f"Error happens when removing {socket_path} : {error}"`

Reply:
The socket file can only be removed when both sides close the socket so one side is going to fail. This shouldn't be treated as an error.